Unit_Tests/src/test/java/homework/transport/RouterTest.java

```diff
@@ -0,0 +1,21 @@
+package homework.transport;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class RouterTest {
+    Router router;
+
+    @Before
+    public void setUp() {
+        router = new Router();
+        router.setStartCoordinates(1.0, 1.0);
+    }
+
+    @Test
+    public void tstPositiveGetDistance() throws Exception {
```

Comment:
This is useless as it tests only 1 case. There are no tests for:
1/ NaN
2/ Negative coordinate
3/ Not initialized state
4/ Two points are the same
etc